rowboat/plugins/core.py

```diff
@@ -29,17 +29,22 @@
 from rowboat.models.notification import Notification
 from rowboat.plugins.modlog import Actions
 
+from yaml import load
```

Comment:
why load this here when you have a `global_config` property in `CorePlugin` -- just move the emoji variables to a property